atomic

```diff
@@ -327,6 +337,18 @@ if __name__ == '__main__':
         epilog="push the latest specified image to a repository.")
     pushp.set_defaults(func='push')
 
+    # atomic pull
+    pullp = subparser.add_parser("pull", help=_("pull latest image from a repository"),
+        epilog="pull the latest specified image from a repository.")
+    pullp.add_argument("--tar", dest="tar", action="store_true",
+                       help=_("import from a Docker save'd tarball."))
+    pullp.add_argument("--docker", dest="docker", action="store_true",
+                       help=_("import from a Docker registry."))
+    pullp.add_argument("--ostree", dest="ostree", action="store_true",
+                       help=_("import from an OSTree repository."))
+    pullp.add_argument("image", help=_("image id"))
+    pullp.set_defaults(func='pull_image')
+
```

Comment:
I am still undecided here about the format.  Should we have --tar/--docker/--ostree or we should rather use a prefix (as we do for install --oci) like:
```
atomic pull tar:///foo/bar.tar
atomic pull ostree://foo:bar
```